static/app/views/eventsV2/results.tsx

```diff
@@ -391,6 +391,25 @@ class Results extends Component<Props, State> {
     }
   };
 
+  handleIntervalChange = (value: string) => {
+    const {router, location} = this.props;
+
+    const newQuery = {
+      ...location.query,
+      interval: value,
+    };
+
+    router.push({
+      pathname: location.pathname,
+      query: newQuery,
```

Comment:
Maybe should consider resetting cursor?

Reply:
Paging cursor right? if so that seems like odd behaviour to me since interval only impacts the table, I'll keep this in mind tho